### ISSUE TITLE
jQuery Datepicker not working within BootSideMenu

### ISSUE THREAD

**Issue by SchweizerSchoggi** [AndreaLombardo/BootSideMenu#28]
Hi Andrea
I am currently integrating BootSideMenu for a flight reservation system. Within the BootSideMenu you can filter bookings by date (from ... to). I am using jQuery Datepicker as my calendar. But unfortunatly the calendar does not show up when I click the textbox. The field is recognized as the trigger field, as it has already received the class "hasDatepicker" dynamically on pageload. Just the calendar does not show up. I already tried different settings for z-index, but no success.

Do you have any idea on this?
Thanks for your help!

Bianca

**Comment by AndreaLombardo** [comment]
Hi Bianca, 
please send me an example of your page.
Maybe you must instantiate the datepicker after bootsidemenu calls...

Let me know ;-)

Andrea

**Comment by SchweizerSchoggi** [comment]
I already tried instantiating the datepickers onOpen and also onBeforeOpen,
but no success.

But now I found the problem right at this moment: I had the wrong ranking
of functions inside my document.ready call. There was the instantiating of
the datepickers first, later on the BSM call. Why it does not help to
re-instantiate the datepickers inside the onOpen event: I don't know. But
at least it's working now, yippieh! ;-)
